transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -33,8 +33,8 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    static {
-        // Ensure JNI is initialized by the time this class is loaded by this time!
+    {
```

Comment:
seems like I learn something new... When is code called that is in an `{}` block but is not `static` @carl-mastrangelo ? Never saw this, even was not aware this is valid java code 😆